cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -193,7 +193,7 @@ static String urlEncode(String s) throws UnsupportedEncodingException {
    */
   static CruiseControlEndPoint getValidEndpoint(HttpServletRequest request, HttpServletResponse response, KafkaCruiseControlConfig config)
       throws IOException {
-    CruiseControlEndPoint endPoint = endPoint(request);
+    CruiseControlEndPoint endPoint = endPoint(request, config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG));
```

Comment:
Passing `config.getString(WebServerConfig.WEBSERVER_API_URLPREFIX_CONFIG)` to endpoint seems redundant (please see my comment on `ParameterUtils#endPoint `)

Reply:
Replied to the previous comment